jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/UpdateChecker.java

```diff
@@ -122,11 +122,13 @@
           return Optional.empty();
         }
         return Optional.of(
-            "A new version of Jib ("
+            "A new version of "
+                + toolName
+                + " ("
                 + version.latest
                 + ") is available (currently using "
                 + currentVersion
-                + "). Update your build configuration to use the latest features and fixes!");
+                + "). Upgrade to use the latest features and fixes!");
```

Comment:
So I think we want to customize the update message. For Jib CLI, it'll really help if it can directly present the GitHub Release for the new version.

Reply:
That does make sense. I modified the message for the jib-cli to include a link to the release page.